homeassistant/components/sensor/recorder.py

```diff
@@ -116,30 +116,22 @@
 
 # Keep track of entities for which a warning about unsupported unit has been logged
 WARN_UNSUPPORTED_UNIT = set()
+WARN_UNSTABLE_UNIT = set()
```

Comment:
Is it OK to have mutable dicts in this scope?

Reply:
No, store in `hass.data`

Reply:
Fixed